Bluejay_Timing.asm

```diff
@@ -257,59 +216,6 @@ calc_next_comm_15deg_set_min:
 
     sjmp    calc_next_comm_period_exit
 
-;**** **** **** **** ****
-; Calculate next commutation period (fast)
-; Fast calculation (Comm_Period4x_H less than 2)
-calc_next_comm_period_fast:
-    ; Calculate new commutation time
-    mov Temp3, Comm_Period4x_L      ; Comm_Period4x holds the time of 4 commutations
-    mov Temp4, Comm_Period4x_H
-
-    ; Divide by 16 and store in Temp5
-    Divide_12Bit_By_16  Temp4, Temp3, Temp5
-
-    clr C
-    mov A, Temp3                    ; Subtract a fraction
-    subb    A, Temp5
-    mov Temp3, A
-    mov A, Temp4
-    subb    A, #0
-    mov Temp4, A
-
-    ; Note: Temp2 is assumed to be zero (approx. Comm_Period4x_H / 4)
```

Comment:
This assumption was not correct. New commutation time H can be different to zero

Reply:
Perfect, thank you.